packages/hub/src/scripts/buildRecentModelRevision/main.ts

```diff
@@ -0,0 +1,182 @@
+import { PrismaClient } from "@prisma/client";
+import { spawn } from "child_process";
+
+import { NotFoundError } from "../../graphql/errors/NotFoundError";
+
+const TIMEOUT_SECONDS = 60; // 60 seconds
+
+const prisma = new PrismaClient();
+
+type SquiggleResult = {
+  errors?: string;
+};
+
+async function runWorker(
+  revisionId: string,
+  code: string,
+  seed: string,
+  timeoutSeconds: number
+): Promise<SquiggleResult> {
+  return new Promise((resolve, _) => {
+    console.log("Spawning worker process for Revision ID: " + revisionId);
+    const worker = spawn(
+      "tsx",
+      ["src/scripts/buildRecentModelRevision/worker.ts"],
+      {
+        stdio: ["pipe", "pipe", "pipe", "ipc"],
+      }
+    );
+
+    const timeoutId = setTimeout(() => {
```

Comment:
I tried using the timeout param in spawn() options, but couldn't get it to log the right error messages. Can go back.